### DISCUSSION TITLE
looks great but can't get it to run.

### DISCUSSION THREAD

**Discussion by gsutton57** [melih-unsal/DemoGPT#14] [Q&A]
PS C:\DemoGPT> streamlit run src/prompt_based/app.py
streamlit : The term 'streamlit' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ streamlit run src/prompt_based/app.py
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (streamlit:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

I have added all the paths, I think?  Is there some extra setup that is needed before running?  I have also tried running it is codespace and got this error 
@gsutton57 ➜ /workspaces/DemoGPT (main) $ streamlit run src/prompt_based/app.py
bash: streamlit: command not found
@gsutton57 ➜ /workspaces/DemoGPT (main) $ dir
CODE_OF_CONDUCT.md  CONTRIBUTING.md  LICENSE  README.md  assets  docs  requirements.txt  src
@gsutton57 ➜ /workspaces/DemoGPT (main) $

**Comment by melih-unsal** [comment]
Hi @gsutton57 , sorry to hear you’re having installation issues!

It is possible that your system has two Python installations one with Anaconda and one directly, so when you are pip installing Streamlit, it is being installed in the site package of python rather than Anaconda’s Python interpreter. 
It could be possible if your environment is not properly activated. you can check “pip --version” and it will point out from where this pip is running. 
If it shows the path to the Python interpreter from Python and not from Anaconda then it shows that streamlit is installed by the standard python interpreter and you are probably trying to use Anaconda's python interpreter to run streamlit.

Could you run `where pip` and `where python` to see if they point out the same parent folder?

**Comment by JA-26-01** [comment]
Hi @gsutton57 . Sorry to see if you are having an installation issue. Also regarding the paths, I think that should not be a problem since the latest Anaconda version has obliterated the inclusion of paths in the environmental variable for the admin use version. I would suggest that you try creating a virtual environment and then try installing streamlit there - using the anaconda prompt will be easier for that.  As @melih-unsal said, it might be due to a conflicting python version issue. I guess using a virtual environment might help in tackling that.
